### ISSUE TITLE
Maybe repharse your pull request text

### ISSUE THREAD

**Issue by LokiMidgard** [JTOne123/GitHubMassUpdater#6]
Your pull request says

> I noticed that your repository and Nuget package are important for our .NET community, but you still haven't enabled SourceLink.

For me this sounds a little misleading. It sounds like you found this package walking a dependency tree of other important packages. And if I understand your readme correct, that's not how you find packages. Your definition of important for the ecosystem seems to be that it is only need to be on Nuget.

I would propably go in a more neutral still positive direction. Maybe something like this:

> Your Project has not enabled SourceLink yet. Help those consuming your package by enabling it.

You probably shouldn't copy that, it still sounds clunky. But it should help understand the intention I have with this issue.

**Comment by JTOne123** [comment]
Agree, but I didn’t discuss it with no one before start so it is reason why text is not perfect 

Feel free to contribute :) this text is on github also ;)
